### PR TITLE
fix(err-msg): #7 - Missing app name message modified.

### PR DESCRIPTION
**Pull request by shangan23**
Bug: #7 

The error message for the missing app name has been modified like below,

```
Please specify the app name:
react-chef <app-name>

For example:
npx react-chef my-app-name

Error: App name is missing.
```

### CODE REVIEW

File: recipes/index.js [hunk 1]
```diff
@@ -1,295 +1,302 @@
...
 
 if (!appName) {
-  error("App name is mandatory to create your boiler plate");
+  console.error('Please specify the app name:')
+  // multiline erro - do not change the indentation
+  log(`${chalk.cyan('npx react-chef')} ${chalk.green('<app-name>')}
+
+For example:
+${chalk.cyan('npx react-chef')} ${chalk.green('my-app-name')}
+`)
```
Comment (ramsunvtech) [inline recipes/index.js]: can we keep all content in one variable and use below method for indentation.

```
const line = (code, noOfSpace = 2, addEmptyLine = true) => {
  return `${" ".repeat(noOfSpace)}${code}${addEmptyLine ? "\n" : ""}`;
};
```

File: recipes/index.js [hunk 2]
```diff
@@ -1,295 +1,302 @@
...
+program.parse(process.argv)
 
 // Create App Directory.
-const appName = program.args[0];
+const appName = program.args[0]
 
 if (!appName) {
-  error("App name is mandatory to create your boiler plate");
+  console.error('Please specify the app name:')
+  // multiline erro - do not change the indentation
```
Comment (ramsunvtech) [inline recipes/index.js]: it has typo `erro`. can we remove this comment. this comment not required when we have below line function

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: recipes/index.js (hunk 1 above)
Comment (shangan23) [reply recipes/index.js]: PR: #10 in this the utility has been implemented

**Comment by shangan23** [comment]
Invalid PR

**Pull request closed without merging**